Header HEADER
string NAME

uint8 GET_STATE

# 1 : START DONE, 2: START CHECKSUM ERROR, 3: START LINE ERROR, 4: START DISCHARGE ERROR, 5: START TIME ERROR
# 6 : STOP DONE, 7: STOP CHECKSUM ERROR
# 8 : WAITING FINISH, 9 : FINISH
# 10 : REQUEST SAVED DATA, 11 : RECEIVE SAVED DATA

int16 N_SAVED_DATA
int16 I_SAVED_DATA
int16[100] GET_LINE
string[100] GET_MSMT_TIME
int16[100] GET_FLOW
int16[100] GET_CAP_TIME
float32[100] GET_INTEG_FLOW
